src/mapping/config/MappingConfiguration.cpp

```diff
@@ -224,9 +258,11 @@ MappingConfiguration::MappingConfiguration(
 
   auto attrSolverRtol = makeXMLAttribute(ATTR_SOLVER_RTOL, 1e-9)
                             .setDocumentation("Solver relative tolerance for convergence");
-  // TODO: Discuss whether we wanto to introduce this attribute
-  // auto attrMaxIterations = makeXMLAttribute(ATTR_MAX_ITERATIONS, 1e6)
-  //                              .setDocumentation("Maximum number of iterations of the solver");
+  // TODO: Discuss whether we wanto to introduce this attribute // TODO: Greedy
+  auto attrMaxIterations = makeXMLAttribute(ATTR_MAX_ITERATIONS, 1e6)
+                               .setDocumentation("Maximum number of iterations of the solver");
+  auto attrgreedySubType = makeXMLAttribute(ATTR_GREEDY_SUBTYPE, "P-cholesky")
```

Comment:
At the moment, it's not entirely clear which version we later on add it preCICE and which not. However, you could use here `add_options()` (see e.g., the constraints) and pass already the available greedy methods. The options also show up automatically in the xml docs then.